Client/core/DXHook/CProxyDirect3D9.cpp

```diff
@@ -166,6 +168,10 @@ HRESULT CProxyDirect3D9::CreateDevice(UINT Adapter, D3DDEVTYPE DeviceType, HWND
     SetWindowTextW(hFocusWindow, MbUTF8ToUTF16("MTA: San Andreas").c_str());
     #endif
 
+    // Set dark titlebar if needed
+    BOOL darkTitleBar = GetSystemRegistryValue((uint)HKEY_CURRENT_USER, "Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize", "AppsUseLightTheme") == "\x0";
```

Comment:
You don't have to check this, because applying `TRUE` will just tell WDM to respect the user's setting (auto/dark/light mode).

Reply:
I think you're wrong. For me with a light theme, the title bar remains dark. From what I know, ``DwmSetWindowAttribute`` sets attributes independently of the theme, doesn't it?

![image](https://github.com/user-attachments/assets/9d8bda14-1dfd-4d30-8304-e363cee49d53)

Reply:
When I tested it the last time, it was working as advertised.

> **DWMWA_USE_IMMERSIVE_DARK_MODE**
Use with [DwmSetWindowAttribute](https://learn.microsoft.com/en-us/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindowattribute). Allows the window frame for this window to be drawn in dark mode colors when the dark mode system setting is enabled. For compatibility reasons, all windows default to light mode regardless of the system setting. The pvAttribute parameter points to a value of type BOOL. TRUE to honor dark mode for the window, FALSE to always use light mode.
> 
> This value is supported starting with Windows 11 Build 22000.  

https://learn.microsoft.com/en-us/windows/win32/api/dwmapi/ne-dwmapi-dwmwindowattribute

Reply:
In that case, I don't know why, but that's how it works. TRUE always means dark, and FALSE always means light, at least on my system. Anyway, since there's a chance (or it's the current behavior), it's better to keep such a check. It's not expensive and helps avoid a potential issue with an incorrect title bar theme

Reply:
Yeah, you're right.